### DISCUSSION TITLE
Welcome to Greek-Sentiment-Analysis Discussions!

### DISCUSSION THREAD

**Discussion by NKryst** [NKryst/Greek-Sentiment-Analysis#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by oldbird1298** [comment]
Καλησπέρα,

Είναι εφικτό ή μήπως έχετε δοκιμάσει το λεξικό (Greek Lexicon) να το περάσεις σε μορφή NRC NRC Word-Emotion Association Lexicon (aka EmoLex) σε γλώσσα R  με την βιβλιοθήκη syuzhet.

Παράδειγμα
get_nrc_sentiment(angling_comments, lexicon = new_dictionary) :

**Reply by NKryst** [reply]
Καλησπέρα, 

Δεν το έχω δει το NRC θα το ελέγξω και θα απαντήσω εδώ ξανά

**Reply by NKryst** [reply]
Καλησπέρα ναι θα μπορούσε να γίνει με εξαίρεση τη στήλη trust

**Comment by oldbird1298** [comment]
Καλησπέρα

Το λεξικό σε μορφή NRC NRC Word-Emotion Association Lexicon (aka EmoLex) είναι αυτό που χρησιμοποιεί η βιβλιοθήκη syuzhet σε γλώσσα R. Είναι πολύ γνωστή και θα ήταν καλή προσπάθεια να παραχθεί ένα nrc για ελληνικά. Θα σου προωθήσω και τα μηνύματα που έχω ανταλλάξει με τον κ. Αδάμ Τσακαλίδη που έχουν παράξει το GRAfs. Εάν πάντως θες να δεις την μορφή του λεξικού φορτώνοντας την βιβλιοθήκη στο R Studio μπορείς να καλέσεις την συνάρτηση
get_nrc_dictionary("nrc") και θα σου την γυρίσει σαν data frame



Ευχαριστώ εκ των προτέρων!!!

Gerontopoulos Dimitrios

***@***.***

***@***.***

Larisa, Thessaly, Greece

Δημήτριος Γεροντόπουλος
***@***.***
________________________________

**Reply by NKryst** [reply]
Καλως μπορώ να το δω
